src/content/docs/dev/send-tokens/introduction.mdx

```diff
@@ -1,171 +1,10 @@
-# Send Tokens Cross-Chain
+# Overview
+Axelar offers cross-chain ERC20 token transfer in two main forms. Gateway tokens and Interchain Token Service based tokens. For general users looking to integrate their own tokens it is recommmended to use ITS. For Dapps looking to make use of more universally recognized assets such as `axlUSDC` or `axlETH`, Gateway tokens serve as a simple solution.
```

Comment:
I think the "more universally recognized" is a judgement that we should leave out, especially as we don't plan to do a lot more new gateway tokens.